app/build.gradle

```diff
@@ -165,6 +166,7 @@ dependencies {
     androidTestCompile "com.android.support.test.espresso:espresso-core:$espressoVersion"
     androidTestCompile "com.android.support.test.espresso:espresso-idling-resource:$espressoVersion"
     androidTestCompile "com.squareup.okhttp3:mockwebserver:$okhttpVersion"
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
```

Comment:
How weird, I didn't see the PR notifications. Sorry!

Any reason why you specified `jre7`? Is it to ensure compatibility with jre7?